### ISSUE TITLE
Bibox exchange does not support fetchTickers

### ISSUE THREAD

**Issue by forexhill** [ccxt/ccxt#2203]
Bibox exchange does not support fetchTickers as can be seen from https://github.com/Biboxcom/api_reference/wiki/api_reference as well as by testing (an empty array is returned whereas fetchTicker works fine). Thus;

`'fetchTickers' => true,` to be changed to `'fetchTickers' => false,` in ccxt/php/bibox.php

Thank you

**Comment by kroitor** [comment]
```
mbp:ccxt igorkroitor$ php -f examples/php/cli.php bibox fetch_tickers
./keys.local.json
bibox->fetch_tickers ()
Array
(
    [BIX/BTC] => Array
        (
            [symbol] => BIX/BTC
            [timestamp] => 1520656397
            [datetime] => 1970-01-18T14:24:16.397+00:00
            [high] => 6.021E-5
            [low] => 5.155E-5
            [bid] =>
            [ask] =>
            [vwap] =>
            [open] =>
            [close] =>
            [first] =>
            [last] => 5.681E-5
            [change] =>
            [percentage] => -5.36%
            ...
```

It does work normally for me. Try it yourself with the script above ([examples/php/cli.php](https://github.com/ccxt/ccxt/blob/master/examples/php/cli.php)). Let us know if it does not help. Thx!

**Comment by forexhill** [comment]
Hey @kroitor - I sent you an email at your gmail, please check.

**Comment by forexhill** [comment]
```$ticker=$exchange->fetchTickers(array('BTC/USDT','BTC/LTC'));var_dump($ticker)``` should work correctly - right? It is this one that returns an empty array. The CLI example works, but seems to fetch all tickers. The code suggest that $symbol can be passed (as an array).

**Comment by kroitor** [comment]
> $ticker=$exchange->fetchTickers(array('BTC/USDT','BTC/LTC'));var_dump($ticker) should work correctly - right?

Right, however...

>  It is this one that returns an empty array.

There are two errors:

1. There was a bug in ccxt that I fixed in version 1.11.62, please update it to make it work properly with BTC/USDT
2. There's a bug in your code as well, as Bibox does not have a "BTC/LTC" symbol, it has "LTC/BTC".

So, if you update and run 

```PHP
$tickers=$exchange->fetchTickers(array('BTC/USDT','BTC/LTC'));
print_r($tickers);
```

It should show just one symbol (the one of the two that is a proper one, namely BTC/USDT):

```
Array
(
    [BTC/USDT] => Array
        (
            [symbol] => BTC/USDT
            [timestamp] => 1520662985
            [datetime] => 1970-01-18T14:24:23.985+00:00
            [high] => 9487.9656
            [low] => 8417.1545
            [bid] =>
            [ask] =>
            [vwap] =>
            [open] =>
            [close] =>
            [first] =>
            [last] => 9458.419
            [change] =>
            [percentage] => +9.96%
            [average] =>
            [baseVolume] =>
            [quoteVolume] =>
            [info] => Array
                (
                    [id] => 36
                    [coin_symbol] => BTC
                    [currency_symbol] => USDT
                    [last] => 9458.4190
                    [high] => 9487.9656
                    [low] => 8417.1545
                    [change] => +856.9624
                    [percent] => +9.96%
                    [vol24H] => 159
                    [amount] => 893331.73
                    [last_cny] => 59966.38
                    [high_cny] => 60153.70
                    [low_cny] => 53364.76
                    [last_usd] => 9458.42
                    [high_usd] => 9487.97
                    [low_usd] => 8417.15
                )

        )

)
```

... and if you fix your code for `"BTC/LTC"` → `"LTC/BTC"`, it will output both tickers as it should.

> The code suggest that $symbol can be passed (as an array).

Yep, an array of $symbol**s** to be precise...

Let us know if it does not help. Thx!
